### ISSUE TITLE
manual-edits.json file is not being detected or recognized by the editor

### ISSUE THREAD

**Issue by rohittcodes** [tscircuit/tscircuit.com#388]
![Image](https://github.com/user-attachments/assets/01d88b56-e1a4-41d9-ae76-386e85bc129d)

**Comment by Bhavyajain21** [comment]
Do we have this file in the code base? @imrishabh18 / @RohittCodes

**Comment by imrishabh18** [comment]
@Bhavyajain21 Yes, here is the template which is used for the `manual-edits.json` [src/lib/templates/manual-edits-template.ts](https://github.com/tscircuit/snippets/blob/e6f7c79d053095bf9b16de812194ec18de0d2094/src/lib/templates/manual-edits-template.ts)
